src/api-service/__app__/onefuzzlib/task_event.py

```diff
@@ -16,7 +17,7 @@ class TaskEvent(BASE_TASK_EVENT, ORMMixin):
     @classmethod
     def get_summary(cls, task_id: UUID) -> List[TaskEventSummary]:
         events = cls.search(query={"task_id": [task_id]})
-        events.sort(key=lambda e: e.Timestamp)
+        events.sort(key=lambda e: e.Timestamp or datetime.datetime.max)
```

Comment:
Maybe add a comment noting that we expect to never reach the rhs of the `or`, and that it doesn't matter if we do.